### ISSUE TITLE
when continuing to hold in a direction should start moving in said direction

### ISSUE THREAD

**Issue by omenking** [omenking/swap-n-pop#1]
In TA when you press and hold the direction key, it will move immediately and pause and then start moving in that direction as long as you hold.

We need to capture the game at 60fps, so we can get the frame data so we can match how the movement works.

**Comment by omenking** [comment]
In order to create automatic movement. We need to modify the `playfield_cursor's` directions.
Each direction passes in `tick` which represent the how long a key has been held:

```js
    left(tick) {
      if (tick > 0) { return }
      game.sounds.select()
      if (this.x > 0) { this.x--; }
    }
```

so if there is tick of 20 that means the key has been held for 20 frames.


- [ ] should automically move if you hold the key long enough
- [ ] find out exact framedata to match same initial delay and velocity of original game
- [ ] code in a way to skip delay if you were to press in rapid opposite direct before moving.

**Comment by Skytrias** [comment]
This has been implemented already. The only thing missing is the frame data. Either omenking or I should look into the frame data to look at the exact behaviour of the Cursor. right now the cursor moves once instantly and then waits a set amount (15 frames right now) until it starts repeating the movement each 15 frames.

**Comment by omenking** [comment]
When I checked into the framedata it would literally move to the next frame if key was held.
I thought there was more of a delay to it but not according to what I captured.

**Comment by Skytrias** [comment]
functionality has been implemented, framedata discussed here #164
